Add jfla ref (#108)
coq-community/hydra-battles: doc/hydras.tex

\SetWatermarkLightness{0.8}
\definecolor{lightorange}{rgb}{1,0.9,0.9}
\SetWatermarkScale{0.35}
\SetWatermarkText{Work Continuously in Progress}
\SetWatermarkColor{lightorange}

\author{Pierre Castéran\\ LaBRI, Univ. Bordeaux, CNRS UMR 5800, \thanks{With contributions by Yves Bertot, \'Evelyne Contejean,  Jérémy Damour, Florian Hatat, Pascal Manoury, Karl Palmskog, Clément Pit-Claudel, and Théo Zimmermann. The formalization of primitive recursive functions was originally authored by 
Russel O'Connor\cite{OConnor05}.}}
\date{\today}
%\date{August 2021}

> \title{Hydras \& Co.\\ A library in Coq of entertaining formal mathematics}

Reviewer: ```suggestion
\title{Hydras \& Co.\\ Formalized mathematics in \coq for inspiration and entertainment}
```

This document is \emph{not} an introductory text for \coq, and there are many aspects of this proof assistant that are not covered. 
 The reader should already have some basic experience with the \coq system. The Reference Manual and several tutorials are available on the \coq website~\cite{Coq}. The first chapters of textbooks like \emph{Interactive Theorem Proving and Program Development}~\cite{BC04}, \emph{Software Foundations}~\cite{SF} or  \emph{Certified Programming with Dependent Types} ~\cite{chlipalacpdt2011} will give you the right background.

 
 

>  \subsection{Making and maintaining these  libraries and book}

Reviewer: ```suggestion
 \subsection{Developing and maintaining the \coq code and documentation}
```

 \textbf{TODO} introductory text and link to \cite{jfla2022}
 

>  \subsubsection{CI/CD}

Reviewer: ```suggestion
 \subsubsection{Continuous Integration and Continuous Delivery}
```
